cloudhub-2/modules/ROOT/pages/ps-create-configure-tgw.adoc

```diff
@@ -166,6 +166,42 @@ To test the connection, use the Network Tools application.
 
 For download and usage information about the Network Tools application, see https://help.mulesoft.com/s/article/How-To-Use-Network-Tools-Application[How To Use Network Tools Application^] (KB article).
 
+[[best-practices]]
+== Maintain the Transit Gateway 
+
+After you test the connection to your private space, periodically ensure your transit gateway connection is maintained. 
+
+If your private space loses access to the transit gateway, the state of the transit gateway is changed to `unknown` or `deleted`. If the transit gateway remains in the `unknown` state for 14 days, Anypoint Platform may remove the private space attachments from the transit gateway. However, the transit gateway itself is not removed. 
```

Comment:
is the transit gateway remains in either `unknown` or `deleted` for 14 days, attachments can be removed

Reply:
```suggestion
If your private space loses access to the transit gateway, the state of the transit gateway is changed to `unknown` or `deleted`. If the transit gateway remains in the `unknown` or `deleted` state for 14 days, Anypoint Platform may remove the private space attachments from the transit gateway. However, the transit gateway itself is not removed. 
```